service/internal/security/crypto_provider.go

```diff
@@ -1,30 +1,8 @@
 package security
 
-import (
-	"crypto"
-	"crypto/elliptic"
-)
-
 const (
 	// Key agreement along P-256
 	AlgorithmECP256R1 = "ec:secp256r1"
 	// Used for encryption with RSA of the KAO
 	AlgorithmRSA2048 = "rsa:2048"
 )
```

Comment:
would it make sense to move these to standard_crypto.go and remove this file?

Reply:
I'll file a follow up ticket that we should export these from the trust package.